apps/main/src/dex/types/main.types.ts

```diff
@@ -18,7 +18,7 @@ export type NetworkConfigFromApi = {
 }
 
 export type NetworkUrlParams = { network: INetworkName }
-export type PoolUrlParams = NetworkUrlParams & { pool: string; formType?: RFormType }
+export type PoolUrlParams = NetworkUrlParams & { poolIdOrAddress: string; formType?: RFormType }
```

Comment:
Wanted to be explicit here, and it helped me pinpoint where `usePoolIdByAddressOrId` should be called 😄